2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_init.py:_log_setup():371] Logging user logs to /home/zinzinbin/codes_for_study/gnn/MNIST_GNN/wandb/run-20220120_173801-3jjm6qks/logs/debug.log
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zinzinbin/codes_for_study/gnn/MNIST_GNN/wandb/run-20220120_173801-3jjm6qks/logs/debug-internal.log
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_init.py:init():404] calling init triggers
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-20 17:38:01,697 INFO    MainThread:9874 [wandb_init.py:init():460] starting backend
2022-01-20 17:38:01,697 INFO    MainThread:9874 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-20 17:38:01,703 INFO    MainThread:9874 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 17:38:01,706 INFO    MainThread:9874 [backend.py:ensure_launched():221] started backend process with pid: 9902
2022-01-20 17:38:01,707 INFO    MainThread:9874 [wandb_init.py:init():469] backend started and connected
2022-01-20 17:38:01,709 INFO    MainThread:9874 [wandb_init.py:init():533] updated telemetry
2022-01-20 17:38:01,718 INFO    MainThread:9874 [wandb_init.py:init():563] communicating current version
2022-01-20 17:38:02,290 INFO    MainThread:9874 [wandb_init.py:init():568] got version response 
2022-01-20 17:38:02,290 INFO    MainThread:9874 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 17:38:02,540 INFO    MainThread:9874 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 17:38:03,690 INFO    MainThread:9874 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 17:38:03,690 INFO    MainThread:9874 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-20 17:38:03,690 INFO    MainThread:9874 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-20 17:38:03,692 INFO    MainThread:9874 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 17:38:03,692 INFO    MainThread:9874 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 17:38:51,246 INFO    MainThread:9874 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-20 17:38:51,247 INFO    MainThread:9874 [wandb_run.py:_restore():1752] restore
2022-01-20 17:38:53,375 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 872
  total_bytes: 872
}

2022-01-20 17:38:53,674 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 872
  total_bytes: 872
}

2022-01-20 17:38:54,553 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 872
  total_bytes: 12686
}

2022-01-20 17:38:54,654 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 872
  total_bytes: 12686
}

2022-01-20 17:38:54,755 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 872
  total_bytes: 12686
}

2022-01-20 17:38:54,856 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:54,957 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:55,058 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:55,159 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:55,260 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:55,361 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:55,719 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}

2022-01-20 17:38:56,223 INFO    MainThread:9874 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 12686
  total_bytes: 12686
}
local_info {
}

2022-01-20 17:38:57,488 INFO    MainThread:9874 [wandb_run.py:_append_files():2180] logging synced files
